README.md

```diff
@@ -36,7 +36,7 @@ rosinstall --catkin . uwrt_mars_rover/arm64_upstream_dependencies.rosinstall
 Commands to update dependencies:
 ```bash
 cd <catkin workspace location>/src
-rosinstall --catkin . uwrt_mars_rover/upstream_dependencies.rosinstall uwrt_mars_rover/metapackage_dependencies.rosinstall
+sudo rosinstall --catkin . uwrt_mars_rover/upstream_dependencies.rosinstall uwrt_mars_rover/metapackage_dependencies.rosinstall
```

Comment:
this cmd fails without sudo

Reply:
iirc u should not need sudo. try on a clean copy of the repo? it may be a conflict with an old version of the file or something. look at the permissions of the existing rosintall file. u can try to chown it